GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -62,27 +62,30 @@
  * It contains the texture displayed as well as information like the points position
  * or the collision mask.
  *
- * @memberof gdjs
+ * @memberOf gdjs
  * @class SpriteAnimationFrame
  * @param {gdjs.ImageManager} imageManager The game image manager
- * @param {FrameData} frameData The frame data used to initialize the frame
+ * @param {SpriteFrameData} frameData The frame data used to initialize the frame
  */
 
 gdjs.SpriteAnimationFrame = function(imageManager, frameData)
 {
     /** @type {string} */
     this.image = frameData ? frameData.image : ""; //TODO: Rename in imageName, and do not store it in the object?
-    /** @type {PIXI.Texture} */
+    /**
+     * @type {PIXI.Texture}
+     * @note need to fix type
```

Comment:
Let's not add "note" for things like this. Either a TODO or better: just let's remove the @type entirely for now.